actors/badge/src/main/java/org/sunbird/badge/util/BadgeAssertionValidator.java

```diff
@@ -0,0 +1,73 @@
+package org.sunbird.badge.util;
+
+import java.util.List;
+import java.util.Map;
+import org.apache.commons.collections.CollectionUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.sunbird.badge.model.BadgeClassExtension;
+import org.sunbird.badge.service.BadgeClassExtensionService;
+import org.sunbird.badge.service.impl.BadgeClassExtensionServiceImpl;
+import org.sunbird.cassandra.CassandraOperation;
+import org.sunbird.common.exception.ProjectCommonException;
+import org.sunbird.common.models.response.Response;
+import org.sunbird.common.models.util.BadgingJsonKey;
+import org.sunbird.common.models.util.JsonKey;
+import org.sunbird.common.responsecode.ResponseCode;
+
+/**
+ * Class to provide validation for Assertion of badge
```

Comment:
change Class to provide validation for Assertion of badge to Class to provide  badge assertion validation.